api/src/main/java/io/opencensus/stats/Measure.java

```diff
@@ -86,7 +86,7 @@ private Measure() {
      */
     public static MeasureDouble create(String name, String description, String unit) {
       checkArgument(StringUtil.isPrintableString(name) && name.length() <= NAME_MAX_LENGTH,
-          "Name should be a ASCII string with a length no greater than 256 characters.");
+          "Name should be a ASCII string with a length no greater than 255 characters.");
```

Comment:
Nit: Use the constant value instead of literal value to make things easier to change in the future.

Reply:
Sure, updated.